R/run_randomisation_analysis.R

```diff
@@ -15,8 +15,25 @@
 #' @export
 #' @examples
 #' if(interactive()) {
-#'   run_randomisation_analysis (
-#'     bd                 = basedata$new(cellsizes=c(500,500))
+#'   bd = basedata$new(cellsizes=c(500,500))
+#'   r_data = list(
+#'                 '250:250' = list (r1 = 13758, r2 = 13860),
+#'                 '250:750' = list (r1 = 11003, r2 = 11134),
+#'                )
+#'
+#'   params = list (bd_params = r_data, raster_params = NULL)
+#'   bd$load_data(params)
+#'
+#'   params = list (bd_params = r_data, raster_params = NULL)
+#'   bd$load_data(params)
+#'
+#'   bd$run_cluster_analysis (
+#'     calculations = c("calc_endemism_central", "calc_richness", "calc_pd"),
+#'     tree = some_phylo_tree
```

Comment:
Suggest comment this out as we don't create a tree in the vignette and it is not a required arg

Reply:
Should be done now